server/src/main/scala/org/http4s/server/middleware/Timeout.scala

```diff
@@ -3,8 +3,8 @@ package server
 package middleware
 
 import cats.data.Kleisli
-import cats.effect._
-import cats.implicits._
+import cats.effect.{Concurrent, Timer}
+import cats.syntax.applicative._
```

Comment:
I still don't like these imports, but I'm planning to do a passthrough to undo some of the damage IntelliJ has done, so I'll clean up then.